tests/default_vars.sh

```diff
@@ -580,6 +582,9 @@ export MOM6_REPRO_LA='False'
 export MOM6_THERMO_SPAN='False'
 # no WW3
 export MOM6_USE_WAVES='False'
+## ocean stochastic physics
```

Comment:
My preference is to have no MOM6 flags or settings included in this commit rather than commenting them out.  Please either remove the optional flags you've added from the default_vars, edit_inputs and MOM_input templates or leave them present but ensure they are completely set false (or off).

Reply:
I agree with @DeniseWorthen, at least these setting shouldn't be presented in default_vars.sh